frontend/tests/components/search/SaveSearchSelector.test.tsx

```diff
@@ -176,7 +174,7 @@ describe("SaveSearchSelector", () => {
       />,
     );
 
-    expect(clientFetchMock).toHaveBeenCalledTimes(3);
```

Comment:
@doug-s-nava I had to update "SaveSearchSelector" to address linting warnings regarding missing dependencies on "useCallback" instances. I'm not sure why adding those dependencies would affect the amount of times these mock fetches are called.